docs/imviz/plugins.rst

```diff
@@ -149,7 +149,11 @@ an interactively selected region. A typical workflow is as follows:
    If this field is not applicable for you, leave it at 0.
    **This field resets every time Data selection changes.**
 8. If you also want photometry result in magnitude unit, you can enter a flux
-   scaling factor in the :guilabel:`Flux scaling` field. The value must be in the
+   scaling factor in the :guilabel:`Flux scaling` field.
+   :guilabel:`Flux scaling` is populated for JWST images
+   if "bunit_data" is a key in the `meta` attribute of the image. The value used
```

Comment:
I suggest change to wording that explains the *intent* instead of the *how*.
```suggestion
   if MJy/sr data unit is detected and pixel area is given to factor out the per-steradian unit. The value used
```

Reply:
p.s. RTD failure would go away if you accept this suggestion.